File: backend/SKyber/src/main/java/edu/cit/SKyber/Entity/Scholarship.java
package edu.cit.SKyber.Entity;



import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Scholarship {
    @Id
    private String id;
    private String title;
    private String description;
    private String link;  
    private String contactEmail; 
    private String type;  
    private String scholarImage;
}

File: frontend_mobile/app/src/main/java/com/example/skyber/dataclass/Scholarship.kt
package com.example.skyber.dataclass

import android.os.Parcelable
import com.google.firebase.database.IgnoreExtraProperties
import kotlinx.parcelize.Parcelize

@Parcelize
@IgnoreExtraProperties
data class Scholarship (
    val id: String = "",
    val title: String = "",
    val description: String = "",
    val link: String ="",
    val contactEmail: String = "",
    val type: String = "",//Private or Public or institution-type Agnostic
    val scholarImage: String? = null
): Parcelable